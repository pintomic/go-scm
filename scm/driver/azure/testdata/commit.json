{
  "commitId": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
  "comment": "Fixed Typo",
  "remoteUrl": "https://dev.azure.com/test-organization/test-project/_git/test-repo/commit/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
  "treeId": "b4eecafa9be2f2006ce1b709d6857b07069b4608",
  "author": {
    "name": "The Author",
    "email": "author@nowhere.com",
    "date": "2012-03-06T23:06:50Z",
    "imageUrl": "https://dev.azure.com/test-organization/_apis/GraphProfile/MemberAvatars/aad.MmVhOWE5YWQtNGY3MS03ZGFkLTkxMDktNjZlM2RjOTUyOWUz"
  },
  "committer": {
    "name": "The Committer",
    "email": "committer@nowhere.com",
    "date": "2012-03-06T23:06:50Z",
    "imageUrl": "https://dev.azure.com/test-organization/_apis/GraphProfile/MemberAvatars/aad.MmVhOWE5YWQtNGY3MS03ZGFkLTkxMDktNjZlM2RjOTUyOWUz"
  }
}