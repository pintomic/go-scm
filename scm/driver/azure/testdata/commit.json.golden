{
  "Sha": "7fd1a60b01f91b314f59955a4e4d4e80d8edf11d",
  "Message": "Fixed Typo",
  "Tree": {
    "Sha": "b4eecafa9be2f2006ce1b709d6857b07069b4608"
  },
  "Author": {
    "Name": "The Author",
    "Email": "author@nowhere.com",
    "Date": "2012-03-06T23:06:50Z",
    "Avatar": "https://dev.azure.com/test-organization/_apis/GraphProfile/MemberAvatars/aad.MmVhOWE5YWQtNGY3MS03ZGFkLTkxMDktNjZlM2RjOTUyOWUz"
  },
  "Committer": {
    "Name": "The Committer",
    "Email": "committer@nowhere.com",
    "Date": "2012-03-06T23:06:50Z",
    "Avatar": "https://dev.azure.com/test-organization/_apis/GraphProfile/MemberAvatars/aad.MmVhOWE5YWQtNGY3MS03ZGFkLTkxMDktNjZlM2RjOTUyOWUz"
  },
  "Link": "https://dev.azure.com/test-organization/test-project/_git/test-repo/commit/7fd1a60b01f91b314f59955a4e4d4e80d8edf11d"
}
